druid/src/widget/image.rs

```diff
@@ -25,7 +25,52 @@ use crate::{
     PaintCtx, Rect, RenderContext, Size, UpdateCtx, Widget,
 };
 
-/// A widget that renders an Image
+/// A widget that renders a bitmap Image.
+///
+/// Contains data about how to fill given space and interpolate pixels.
```

Comment:
Lack of articles feels off. Perhaps something like *How to fill **the** given space*